src/main/java/com/github/jrrdev/mantisbtsync/rest/repositories/HandlerStatsRepository.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * 
+ */
+package com.github.jrrdev.mantisbtsync.rest.repositories;
+
+import java.util.List;
+
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import org.springframework.data.rest.core.annotation.RepositoryRestResource;
+
+import com.github.jrrdev.mantisbtsync.rest.domain.HandlerStats;
+import com.github.jrrdev.mantisbtsync.rest.repositories.commons.ReadOnlyPagingAndSortingRepository;
+
+/**
+ * Repository for domain object stored into handlers_stats.
+ * 
+ * @author slopezgarcia
+ *
+ */
+@RepositoryRestResource
+public interface HandlerStatsRepository extends
+		ReadOnlyPagingAndSortingRepository<HandlerStats, Long> {
+
+	@Query(value="SELECT new com.github.jrrdev.mantisbtsync.rest.domain.HandlerStats(CASE WHEN hs.handler.id = :handlerSopra THEN 'Sopra Steria' ELSE 'Minist�re de la Justice' END, hs.project.name, hs.status.name, hs.computeDate, SUM(hs.nbIssues)) "
+			+ "FROM HandlerStats hs WHERE hs.project.id IN :project " 
+			+ "AND hs.status.id NOT IN :status "
+			+ "AND hs.computeDate = (SELECT MAX(hs2.computeDate) FROM HandlerStats hs2) "
+			+ "GROUP  BY CASE WHEN hs.handler.id = :handlerSopra THEN 'Sopra Steria' ELSE 'Minist�re de la Justice' END, hs.project.name, hs.status.name, hs.computeDate "
+			+ "ORDER BY CASE WHEN hs.handler.id = :handlerSopra THEN 'Sopra Steria' ELSE 'Minist�re de la Justice' END, hs.status.name")
+	List<HandlerStats> findStatsByProjectIdAndStatusIdNotInWithSopraHandler(@Param("handlerSopra") Long handlerSopra, @Param("project") Long project, @Param("status") List<Long> status);
```

Comment:
This method is too specific.
I suggest you replace it by a generic one and perform further operations in the code calling this rest endpoint.

For instance you can just add one method to this repository : 
`List<HandlerStats> findStatsByProjectIdAndStatusIdNotIn`

Then in your calling code, call findStatsByProjectIdAndStatusIdNotIn with an order by computeDate desc,handlerId and parse the resulting list to compute the sum.